systemtest/src/test/java/io/strimzi/systemtest/metrics/MetricsST.java

```diff
@@ -699,6 +708,217 @@ void testKafkaMetricsSettings() {
         }
     }
 
+    /**
+     * @description This test case check that enabling metrics and metrics manipulation triggers Rolling Update.
+     *
+     * @steps
+     *  1. - Deploy Kafka Cluster with Zookeeper and with disabled metrics configuration
+     *     - Cluster is deployed
+     *  2. - Change specification of Kafka Cluster by configuring metrics for Kafka, Zookeeper, and configuring metrics Exporter
+     *     - Allowing metrics triggers Rolling Update
+     *  3. - Setup or deploy necessary scraper, metric rules, and collectors and collect metrics
+     *     - Metrics are successfully collected
+     *  4. - Modify patterns in rules for collecting metrics in Zookeeper and Kafka by updating respective Config Maps
+     *     - Respective changes do not trigger Rolling Update, Cluster remains stable and metrics are exposed according to new rules
+     *  5. - Change specification of Kafka Cluster by removing any metric related configuration
+     *     - Rolling Update is triggered and metrics are no longer present.
+     *
+     * @usecase
+     *  - metrics
+     *  - kafka-metrics-rolling-update
+     *  - rolling-update
+     */
+    @IsolatedTest
+    @Tag(ROLLING_UPDATE)
+    @KRaftNotSupported("Zookeeper is not supported by KRaft mode and is used in this test class")
+    @SuppressWarnings("checkstyle:MethodLength")
+    void testMetricsChange(ExtensionContext extensionContext) throws JsonProcessingException {
+        final TestStorage testStorage = new TestStorage(extensionContext, Environment.TEST_SUITE_NAMESPACE);
```

Comment:
You can remove `Environment.TEST_SUITE_NAMESPACE` because now it is encapsulated in the TestStorage logic that if you don't provide a namespace it will automatically assign `Environment.TEST_SUITE_NAMESPACE`